engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/blueprint/NadelExecutionBlueprint.kt

```diff
@@ -16,18 +16,44 @@ import graphql.schema.GraphQLSchema
 data class NadelOverallExecutionBlueprint(
     val schema: GraphQLSchema,
     val fieldInstructions: Map<FieldCoordinates, List<NadelFieldInstruction>>,
+    val underlyingTypeNameToOverallNameByService: Map<Service, Map<String, String>>,
+    val overAllTypeNameToUnderlyingNameByService: Map<Service, Map<String, String>>,
```

Comment:
So this is contentious and perhaps I remove it

I started by capturing a per service map of `overall -> underlying` and the reverse in order to build the actual `Set` of overall types per service.

But it fails to pass the tests.  Now we dont need this per se -  but I wanted to show this code before we removed it